invokeai/frontend/web/src/services/api/run-graph.ts

```diff
@@ -135,25 +135,23 @@ export const runGraph = (arg: RunGraphArg): Promise<RunGraphReturn> => {
  * Creates production dependencies for runGraph using Redux store and socket.
  */
 export const buildRunGraphDependencies = (
-  store: AppStore,
+  dispatch: AppDispatch,
```

Comment:
This was typed as needing the whole redux store (`getState`, `subscribe`,  `dispatch`), but it only actually uses `dispatch`.

The dnd handlers are set up to get `dispatch` and `getState` only. So, we needed to change this to not require the whole store, in order to expand prompts via runGraph via dnd handler